packages/react-components/src/components/timegraph-output-component.tsx

```diff
@@ -472,7 +478,9 @@ export class TimegraphOutputComponent extends AbstractTreeOutputComponent<Timegr
                     size: symbolSize,
                     color: color ? color.color : 0x000000,
                     alpha: color ? color.alpha : 1.0,
-                    verticalAlign: vAlign ? vAlign : 'middle'
+                    verticalAlign: vAlign ? vAlign : 'middle',
+                    opacity: color ? color.alpha : 0.2
```

Comment:
There is already support for alpha above, but I notice that it's not part of the TimeGraphAnnotationStyle definition and therefore not used by timeline-chart. It's the same for alpha in TimeGraphStateStyle.

I think we'll need to update this, but needs to be fixed in timeline-chart first.

The default should still be 1.0 if the style from TSP doesn't have a color. That will be fixed at the same time.